### ISSUE TITLE
Replacing Rancher Desktop's built in Trafaik 

### ISSUE THREAD

**Issue by carlskii** [caddyserver/ingress#116]
Hi, 

I might be missing the point here, but it would be super useful to have some documentation that covers using this instead of  Rancher Desktop's (K3D) which includes Trafaik. I'm a little new to K8s but love caddy so would like to standardise on using it everywhere. Also, are there any docs that cover how to configure Caddy in the context of K8s i.e. routing etc?

**Comment by Embraser01** [comment]
Hi,

I'm not too familiar with Traefik but it would be nice to have some "migrate from ..." guide (another one for nginx would be nice too). I'll rename this issue and let it out there for someone to help 

Regarding the configuration of caddy ingress controller, you can check the [values](https://github.com/caddyserver/ingress/blob/master/charts/caddy-ingress-controller/values.yaml) file of the Helm Chart. There is not a lot of documentation yet. 
Basically everything should work as the official [K8S Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/), plus, we support some specific annotations for Ingress specific route options and a global config map for Caddy global options.

**Comment by carlskii** [comment]
Actually thinking this through a little more it's more about a working demo in k3s/k3d rather than it needing to be Rancher Desktop specific.
